pipelines/lib/util.groovy

```diff
@@ -310,6 +310,9 @@ def void runPublish(Map p) {
   if (useP.parameters.SPLENV_REF) {
     jobParameters += string(name: 'SPLENV_REF', value: useP.parameters.SPLENV_REF)
   }
+  if (useP.parameters.OLD_MATRIX) {
+    jobParameters += string(name: 'OLD_MATRIX', value: useP.parameters.OLD_MATRIX)
```

Comment:
Are you sure this should be `string`? (and similar uses below)

Reply:
right, it should be a boolean